### ISSUE TITLE
Is it possible to truncate BPTT to less than max_time in dynamic_rnn?

### ISSUE THREAD

**Issue by 3rd3** [tensorflow/tensorflow#2803]
I am wondering whether it is possible to feed, say, sequences of length 10000 while letting the gradients only flow back over a maximum of 100 time steps. This seem to me a pretty common thing to do in sequential learning tasks, so I am wondering whether there is support for that in `dynamic_rnn` and if not, whether it is planned.

**Comment by ebrevdo** [comment]
We have some internal code that does this, am planning to open source it someone this summer.

**Comment by ebrevdo** [comment]
The code is now opensourced on master branch.  See `tf.contrib.training.batch_sequences_with_states` (sorry; for now the only examples are in the unit tests).  This was added in 955efc90026ee08205ad84551dd4a308439fd741.
